tests/util.go

```diff
@@ -25,13 +31,20 @@ type Result struct {
 }
 
 var KubeVirtOcPath = ""
+var KubeVirtVirtctlPath = ""
+
+const NamespaceTestDefault = "kubevirt-test-default"
```

Comment:
I don't understand why do you want to split the constants, it will only make more lines and does not improve readability

Reply:
I think it really does, refer to kubevirt/tests/util.go.